lib/stdlib/src/lists.erl

```diff
@@ -113,11 +112,10 @@ keyfind(_, _, _) ->
 Returns `true` if there is a tuple in `TupleList` whose `N`th element compares
 equal to `Key`, otherwise `false`.
 """.
--spec keymember(Key, N, TupleList) -> boolean() when
+-spec keymember(Key, N, List) -> boolean() when
       Key :: term(),
       N :: pos_integer(),
-      TupleList :: [Tuple],
-      Tuple :: tuple().
+      List :: [term()].
```

Comment:
Why `term()` and not `tuple()` here?

Reply:
ah, because it is actually allowed to have non-tuples in the list. I did not know that.